### PR TITLE
fix uncached stop words

### PR DESCRIPTION
**Pull request by grimoire**
Fix: The last generated token will not be cached.

### CODE REVIEW

File: lmdeploy/pytorch/engine/engine.py [hunk 1]
```diff
@@ -501,7 +504,8 @@ def update_running(self, running: SeqList, next_token_ids: torch.Tensor,
                 continue
             msg.num_new_tokens += 1
             update_token = token
-            if msg.num_new_tokens > msg.sampling_param.max_new_tokens:
+            # if msg.num_new_tokens > msg.sampling_param.max_new_tokens:
```
Comment (RunningLeon) [inline lmdeploy/pytorch/engine/engine.py:507]: could we remove this commented line?

**Review by RunningLeon — APPROVED** [review]
LGTM

**Comment by lvhan028** [comment]
If a word or a phrase such as "\n\n" is tokenized into multiple token ids, can it act as a stop_word in pytorch engine?

**Comment by lvhan028** [comment]
Before applying this PR, how can we reproduce this issue? We probably need to add it to the testcases

**Comment by grimoire** [comment]
> If a word or a phrase such as "\n\n" is tokenized into multiple token ids, can it act as a stop_word in pytorch engine?

Nope.

**Pull request merged**